### ISSUE TITLE
Missing shmem_signal_wait_until

### ISSUE THREAD

**Issue by dalcinl** [Sandia-OpenSHMEM/SOS#1015]
Looks like `shmem_signal_wait_until` is missing, despite it is documented in the manpages.

```
$ grep -r shmem_signal_wait_until /home/devel/shmem/sandia
/home/devel/shmem/sandia/share/man/man3/shmem_signal_wait_until.3:shmem_signal_wait_until \- 
```

**Comment by dalcinl** [comment]
@davidozog My guess is that this should be easy to implement for you. Could give it a shot?

**Comment by davidozog** [comment]
Oh, interesting.  Thanks for catching that - sure thing.

**Comment by wrrobin** [comment]
Can we close this now? @dalcinl @davidozog

**Comment by davidozog** [comment]
Yes, thanks - closed by #1028
